build.sbt

```diff
@@ -153,14 +153,27 @@ lazy val interopTests = Project(
     }
     )))
 
+lazy val playTestkit = Project(
```

Comment:
Should this be aggregate in `root`?

https://github.com/akka/akka-grpc/blob/2dffcba8f23196eb54bf290676db9f8f6319d420/build.sbt#L252

Reply:
If not, we should invoke the `compile`/`test` tasks on it explicitly in both the travis and jenkins builds.